apps/frontend/core-dapp/src/features/history/HistoryTransaction.tsx

```diff
@@ -0,0 +1,103 @@
+import { getFullDateShortenMonthFromSeconds } from '../../utils/date-utils'
+import { useRootStore } from '../../context/RootStoreProvider'
+import { PositionName } from '../position/Position'
+
+const { toUsd } = numberFormatter
+
+const Label = styled.p`
+  color: ${({ theme }): string => theme.color.neutral1};
+  margin: 0;
+`
+
+const EventLink = styled.a<{ $event: string }>`
+  align-items: center;
+  background: ${({ $event, theme }): string =>
+    theme.color[transactionTypeColors[$event].backgroundColor]};
+  border-radius: ${spacingIncrement(8)};
+  color: ${({ $event, theme }): string => theme.color[transactionTypeColors[$event].color]};
+  display: flex;
+  padding: ${spacingIncrement(2)} ${spacingIncrement(8)};
+`
+
+type HistoryTransactionProps = {
+  event: HistoryTransactionEntity['event']
+  eventType: HistoryTransactionEntity['eventType']
+  iconName: HistoryTransactionEntity['iconName']
+  marketId: HistoryTransactionEntity['marketId']
+  name: HistoryTransactionEntity['name']
+  timestamp: HistoryTransactionEntity['timestamp']
+  transactionHash: HistoryTransactionEntity['transactionHash']
+  usdValue: HistoryTransactionEntity['usdValue']
+}
+const HistoryTransaction: React.FC<HistoryTransactionProps> = ({
+  event,
+  eventType,
+  iconName,
+  marketId,
+  name,
+  timestamp,
+  transactionHash,
+  usdValue,
+}) => {
+  const { web3Store } = useRootStore()
+  const { network } = web3Store
+
+  let nameRedirectUrl: LinkProps['href'] | undefined
+  if (marketId !== undefined && (event === 'Opened' || event === 'Closed'))
+    nameRedirectUrl = {
+      pathname: '/trade',
+      query: {
+        action: event === 'Opened' ? 'open' : 'close',
+        direction: eventType,
+        marketId,
+      },
+    }
+  if (event === 'Deposited') nameRedirectUrl = '/portfolio/deposit'
+  if (event === 'Withdrawn') nameRedirectUrl = '/portfolio/withdraw'
+
+  return (
+    <Flex justifyContent="start" gap={8} width="100%">
+      <Icon name={iconName} height={spacingIncrement(48)} width={spacingIncrement(48)} />
+      <Flex flexDirection="column" alignItems="start">
+        <Flex gap={4}>
+          {nameRedirectUrl ? (
+            <Link href={nameRedirectUrl} passHref>
+              <PositionName as="a">{name}</PositionName>
+            </Link>
+          ) : (
+            <PositionName>{name}</PositionName>
+          )}
+
+          {eventType && <PositionLabel positionType={eventType} />}
+        </Flex>
+        <Label>{toUsd(usdValue)}</Label>
+      </Flex>
+      <Flex flexDirection="column" alignItems="end" ml="auto" gap={6}>
+        <Link href={`${NETWORKS[network.name].blockExplorer}/tx/${transactionHash}`} passHref>
+          <EventLink $event={event} target="_blank" rel="noreferrer noopener nofollow">
+            {event}{' '}
+            <Icon
+              name="arrow-up-right"
+              width={spacingIncrement(14)}
+              height={spacingIncrement(14)}
+            />
```

Comment:
I think can just use the html nearr entity to make it simpler and more readable, wdyt?

Ref: https://www.w3schools.com/charsets/ref_html_entities_n.asp

```suggestion
            {event} &#x02197;
```

Reply:
Nice!